create_toy_data.py

```diff
@@ -62,42 +65,115 @@ def load_data(
                     obj that returns the content
                     for each document.
 
-        num_docs : inn / None
+        num_docs : int / None
                    number of documents to process
                    if None, return all
 
         return_tags : boolean
                       if true, tags are returned w/
                       words in a tuple
 
+        return_sent_labels : boolean
+                            if true, a boolean is
+                            returned for whether the
+                            token repr. EOS.
         Returns
         -------
         data : Python generator
                yields a single token (w/ POS)
     '''
 
+    default_splitter = nltk.data.load(
+        'tokenizers/punkt/english.pickle')
+
```

Comment:
Hmm... well what this tell me is that we need a better labeling system... Maybe we can find a dataset somewhere